ts/Extensions/Annotations/Annotations.ts

```diff
@@ -1419,21 +1380,27 @@ merge<Annotation>(
                  * series - it is referenced by the point's id - or a new point
                  * with defined x, y properties and optionally axes.
                  *
+                 * @sample highcharts/annotations/mock-points/
+                 *         Attach annotation to a mock point
+                 *
                  * @declare   Highcharts.AnnotationMockPointOptionsObject
                  * @type      {string|Highcharts.AnnotationMockPointOptionsObject}
                  * @extends   annotations.labels.point
+                 * @requires  modules/annotations
                  * @apioption annotations.shapes.point
                  */
 
                 /**
-                 * An array of points for the shape. This option is available
+                 * An array of points for the shape
+                 * or a callback function that returns that shape point.
+                 *
+                 * This option is available
                  * for shapes which can use multiple points such as path. A
                  * point can be either a point object or a point's id.
                  *
                  * @see [annotations.shapes.point](annotations.shapes.point.html)
                  *
-                 * @declare   Highcharts.AnnotationMockPointOptionsObject
-                 * @type      {Array<string|*>}
+                 * @type      {Array<Highcharts.AnnotationShapePointOptions>}
```

Comment:
Missing `string` type (it has in the description above: `A point can be either a point object or a point's id.`